.github/workflows/newman.yml

```
@@ -46,7 +48,13 @@
         required: true
       jira_user:
         required: true
-      jira_token_test:
+      jira_api_token:
+        required: true
+      jira_base_url:
+        required: true
+      jira_user_email:
+        required: true
+      jira_api_token:
```

actionlint

key "jira_api_token" is duplicated in "secrets" section. previously defined at line:51,col:7. note that this key is case insensitive

(syntax-check)